worklist/constants.py

```diff
@@ -25,7 +25,7 @@
 SEARCH_BY_USERNAME_FOR_WORKLISTS = 'username'
 
 # Default timeout for HTTP GET Request
-TIMEOUT_IN_SECONDS = 10
+TIMEOUT_IN_SECONDS = 180
```

Comment:
Is this constant used for other GET requests, such as when talking to the Wikipedia API?

We probably only want the larger timeout for PetScan requests specifically, so maybe we need to have two constants here.

Reply:
Thanks, I have made the change and also updated the error message.